stdlib/types.pyi

```diff
@@ -304,6 +304,10 @@ class MappingProxyType(Mapping[_KT, _VT_co]):
     def keys(self) -> KeysView[_KT]: ...
     def values(self) -> ValuesView[_VT_co]: ...
     def items(self) -> ItemsView[_KT, _VT_co]: ...
+    @overload
+    def get(self, key: _KT, /) -> _VT_co | None: ...  # type: ignore[override]
+    @overload
+    def get(self, key: _KT, default: _T2, /) -> _VT_co | _T2: ...  # type: ignore[override]
```

Comment:
```suggestion
    def get(self, key: _KT, default: _VT_co | _T2, /) -> _VT_co | _T2: ...  # type: ignore[override]
```
I don't remember why `Mapping.get` is typed the way it is, but maybe we should match that over here

Reply:
Yeah, I don't think it makes a difference, at least on newer versions of MyPy/Pyright, but probably good to stay consistent for now.

Reply:
Hmm, starting to see some failures now.